### PR TITLE
mnc: init at 0.4

### PR DESCRIPTION
**Pull request by wentam**
###### Description of changes

Echos the time when the next cronjob will be ran. From [here](https://github.com/wentam/sxmo-nix). This is a dependency of [sxmo](https://sxmo.org).

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-linux.pkgsCross.aarch64-multiplatform
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/mnc/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, pkgs
+, lib
+, fetchFromSourcehut
+, buildGoModule
```
Comment (muscaln) [inline pkgs/tools/misc/mnc/default.nix:5]:
```suggestion
{ lib
, buildGoModule
, fetchFromSourceHut
```

File: pkgs/tools/misc/mnc/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+}:
+
+buildGoModule rec {
+  pname = "mnc";
+  version = "0.4";
+  vendorSha256 = "sha256-H0KmGTWyjZOZLIEWophCwRYPeKLxBC050RI7cMXNbPs=";
```
Comment (muscaln) [inline pkgs/tools/misc/mnc/default.nix:11]:
```suggestion

  vendorSha256 = "sha256-H0KmGTWyjZOZLIEWophCwRYPeKLxBC050RI7cMXNbPs=";
```

File: pkgs/tools/misc/mnc/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+  src = fetchFromSourcehut {
+    owner = "~anjan";
+    repo = "mnc";
+    rev = version;
+    sha256 = "sha256-S7MBIxuYI+cc8OMQULt7VS7ouPqhq0Jk+rz6E5GyKac=";
+  };
+
+  meta = with lib; {
+    description = "Opens the user's crontab and echos the time when the next cronjob will be ran.";
```
Comment (muscaln) [inline pkgs/tools/misc/mnc/default.nix:21]:
```suggestion
    description = "Opens the user's crontab and echos the time when the next cronjob will be ran";
```

**Review by muscaln — CHANGES_REQUESTED** [review]

**Comment by wentam** [comment]
Requested changes applied

**Review by muscaln — APPROVED** [review]
LGTM

**Review by Mindavi — APPROVED** [review]

**Pull request merged**